packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx

```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+import { CreateDaoSteps, accordionItems, stepsNames } from './Shared';
+import { useCreateDao, ICreateDaoCollaterals } from './CreateDaoContextProvider';
+import {
+  useLayout,
+  Grid,
+  GridItem,
+  Accordion,
+  TextInput,
+  Switch,
+  Box,
+  Button,
+  StyledText,
+  Steps,
+  Split,
+  Info,
+  IconMinus,
+  IconPlus,
+  IconArrowLeft,
+  SPACING,
+} from '@aragon/ui';
+
+const CreateDaoCollateral: React.FC<{
+  setActiveStep: React.Dispatch<React.SetStateAction<CreateDaoSteps>>;
+}> = ({ setActiveStep }) => {
+  const { layoutName } = useLayout();
+  const spacing = SPACING[layoutName];
+  const { collaterals, setCollaterals } = useCreateDao();
+  const {
+    scheduellAddress,
+    scheduellAmount,
+    isScheduellNewDaoToken,
+    challengeAddress,
+    challengeAmount,
+    isChallengeNewDaoToken,
+    isAnyAddress,
+    executionAddressList,
+  } = collaterals;
+
+  const updateCollaterals = (
+    indexType:
+      | 'scheduellAddress'
+      | 'scheduellAmount'
+      | 'isScheduellNewDaoToken'
+      | 'challengeAddress'
+      | 'challengeAmount'
+      | 'isChallengeNewDaoToken'
+      | 'isAnyAddress'
+      | 'executionAddressList',
+    value: any,
+  ) => {
+    console.log('updateCollaterals', indexType, value);
+    const newCollaterals: ICreateDaoCollaterals = { ...collaterals };
+    (newCollaterals[indexType] as any) = value;
+    setCollaterals(newCollaterals);
+  };
+
+  return (
+    <Grid layout={true}>
```

Comment:
If you can break this into smaller component, will make this code cleaner and easier to read.

Reply:
yes, i have plans to do the split once we have the switcher componer ready